okhttp/api/android/okhttp.api

```diff
@@ -326,6 +326,22 @@ public final class okhttp3/CipherSuite$Companion {
 	public final fun forJavaName (Ljava/lang/String;)Lokhttp3/CipherSuite;
 }
 
+public class okhttp3/CompressionInterceptor : okhttp3/Interceptor {
+	public static final field Companion Lokhttp3/CompressionInterceptor$Companion;
+	public fun <init> ([Lokhttp3/CompressionInterceptor$DecompressionAlgorithm;)V
+	public final fun getAlgorithms ()[Lokhttp3/CompressionInterceptor$DecompressionAlgorithm;
+	public fun intercept (Lokhttp3/Interceptor$Chain;)Lokhttp3/Response;
+}
+
+public final class okhttp3/CompressionInterceptor$Companion {
+	public final fun getGzip ()Lokhttp3/CompressionInterceptor$DecompressionAlgorithm;
```

Comment:
Should this be `@JvmStatic` so Java users don’t need to use the Companion class?